### PR TITLE
Forms Automated Test - LPS-133783 | master

### PR DESCRIPTION
**Pull request by wesleyroberts**

### CODE REVIEW

**Comment by natocesarrego** [comment]
@wesleyroberts, the test created on this pull is actually associated with LPS-133790 and not LPS-133783. Can you please clarify the things on this pull and on this other [pull](https://github.com/liferay-forms/liferay-portal/pull/584)?

Thanks

**Comment by natocesarrego** [comment]
@wesleyroberts, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?

Thanks.

**Comment by wesleyroberts** [comment]
@natocesarrego I sorry, i was wrong the testcase and send in local incorrect.

**Comment by wesleyroberts** [comment]
@natocesarrego  This PR is related  to this other https://github.com/liferay-forms/liferay-portal/pull/584

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase [hunk 1]
```diff
@@ -1478,14 +1478,46 @@ definition {
...
 
+		FormsAdminNavigator.gotoPreviewForm();
+
+		Navigator.selectSpecificWindow(windowTitle = "Form - Forms");
+
+		FormFields.viewTextField(
+			fieldLabel = "Text",
+			fieldName = "Text");
+
+		Form.viewFormPlaceholderText(key_value = "Test Placeholder Text");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase:1508]: @wesleyroberts, can we use the macro `FormFields.viewFieldPlaceholderText` instead?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase [hunk 2]
```diff
@@ -1232,7 +1232,7 @@ definition {
 	@description = "Verify that a Rich Text Field can be previewed"
 	@priority = "5"
 	test RichTextFieldCanBePreviewed {
-		property forms.functional = "false";
+		property forms.functional = "true";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase:1235]: @wesleyroberts, is this change necessary for the test `TextFieldCanBePreviewed` pass?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase (hunk 2 above)
Comment (wesleyroberts) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase]: Changes  made @natocesarrego .

**Comment by natocesarrego** [comment]
ci:forward
